src/components/vertical-divider/vertical-divider.component.tsx

```diff
@@ -101,13 +104,37 @@ type TintRange =
   | 98
   | 99
   | 100;
+
 export interface VerticalDividerPropTypes extends SpaceProps {
   h?: number | string;
+  height?: number | string;
```

Comment:
Good plan adding `height` in, why on earth `h` is the original prop name I have no idea 😅